stairway/src/main/java/bio/terra/stairway/Flight.java

```diff
@@ -1,426 +1,65 @@
 package bio.terra.stairway;
 
-import static bio.terra.stairway.FlightStatus.READY;
-import static bio.terra.stairway.FlightStatus.READY_TO_RESTART;
-import static bio.terra.stairway.FlightStatus.WAITING;
-
-import bio.terra.stairway.exception.DatabaseOperationException;
-import bio.terra.stairway.exception.RetryException;
-import bio.terra.stairway.exception.StairwayExecutionException;
-import java.util.HashSet;
 import java.util.LinkedList;
 import java.util.List;
-import java.util.Set;
-import org.apache.commons.lang3.builder.ToStringBuilder;
-import org.apache.commons.lang3.builder.ToStringStyle;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /**
- * Manage the atomic execution of a series of Steps This base class has the mechanisms for executing
- * the series of steps.
+ * The Flight object is the vehicle Stairway uses for (re)building the steps of a flight in
+ * preparation for running the flight. The object is not used in the actual execution of the flight.
```

Comment:
`The object is not used in the actual execution of the flight.` <- It's not clear to me what that means. Mind adding a little more?